generic3g/VerticalGeom.F90

```diff
@@ -0,0 +1,50 @@
+#include "MAPL_Generic.h"
```

Comment:
Is the intent that this will eventually be subclassed or parameterized for different types of vertical  axes?  (levels, log P, P, ...)?